src/ducks/account/helpers.js

```diff
@@ -21,8 +21,21 @@ export const getAccountInstitutionLabel = account => {
 export const getAccountLabel = account =>
   account ? account.shortLabel || account.label : ''
 
-export const getAccountUpdateDate = account =>
-  get(account, 'cozyMetadata.updatedAt')
+export const getAccountUpdateDate = account => {
+  const vendorUpdatedAt = get(account, 'metadata.updatedAt')
+
+  if (vendorUpdatedAt) {
+    return vendorUpdatedAt
+  }
+
+  const cozyUpdatedAt = get(account, 'cozyMetadata.updatedAt')
+
+  if (cozyUpdatedAt) {
+    return cozyUpdatedAt
+  }
+
+  return null
+}
```

Comment:
Could this be moved to cozy-libs/cozy-doctypes ? It's good to start using this even in the views I think. WDYT ?

Reply:
Yep nice idea. See https://github.com/cozy/cozy-libs/pull/423